crates/krun-server/src/bin/krun-server.rs

```diff
@@ -0,0 +1,31 @@
+use std::env;
+use std::process::Command;
+use std::thread;
+
+use anyhow::Result;
+use krun_server::cli_options::options;
+use krun_server::server::start_server;
```

Comment:
CS: merge imports

```suggestion
use krun_server::{cli_options::options, server::start_server};
```

Reply:
But yeah, we could change this if you'd prefer. The current Rust style guide says:

> Prefer to use multiple imports rather than a multi-line import.

https://github.com/rust-lang/rust/blob/9f2d0b3490729ed65da73f64b28d8d1f197af113/src/doc/style-guide/src/items.md#large-list-imports

Reply:
Let's make this homogeneous in a follow-up PR.

Reply:
#28